src/static/riot/competitions/editor/_leaderboard.tag

```diff
@@ -283,11 +315,11 @@
                 self.leaderboards[self.selected_leaderboard_index] = self.get_leaderboard_data()
             } else {
                 self.leaderboards.push(self.get_leaderboard_data())
-
             }
-            self.close_modal()
             self.form_updated()
+            self.close_modal()
             self.update()
+            self.selected_submission_rule = undefined
```

Comment:
Why do we set this back to `undefined`?

Reply:
This variable is used to store the submission rule in Javascript, because for some reason it isn't being stored in the DOM like the other dropdowns. Once the modal is closed it doesn't need to be saved anymore, and logically it makes sense for it be undefined when there is no modal on the screen.